### PR TITLE
fix(style guide page): copy edits.

### PR DESCRIPTION
**Pull request by mmfred**
updates to released page.

### CODE REVIEW

**Comment by mmfred** [comment]
This is ready for review. Should not have trouble building.

File: src/nav/style-guide.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ pages:
     pages:
       - title: Capitalization
         path: /docs/style-guide/capitalization/capitalization
-      - title: Product capability and feature capitalization and usage
+      - title: Product, capability and feature capitalization and usage
```
Comment (bradleycamacho) [inline src/nav/style-guide.yml:18]:
```suggestion
      - title: Product, capability, and feature capitalization and usage
```

**Review by bradleycamacho — APPROVED** [review]
One small change :)

File: src/nav/style-guide.yml (hunk 1 above)
Comment (mmfred) [reply src/nav/style-guide.yml:18]: I caught that right after I was all ready to go, so now we'll see what happens. I'm going to merge. Thanks!

**Pull request merged**